src/components/Modal/BaseModal.js

```diff
@@ -142,13 +144,13 @@ function BaseModal({
                 {
                     windowWidth,
                     windowHeight,
-                    isSmallScreenWidth,
+                    shouldUseNarrowLayout,
```

Comment:
Lets also update `getModalStyles` i see this function extracting `isSmallScreenWidth`, we changed that. we have to update the name and types.

Reply:
https://github.com/rayane-djouah/App/blob/Create-a-useResponsiveLayout-hook/src/styles/getModalStyles.ts#L48